src/app/app.component.spec.ts

```diff
@@ -30,6 +53,6 @@ describe('AppComponent', () => {
     const fixture = TestBed.createComponent(AppComponent);
     fixture.detectChanges();
     const compiled = fixture.debugElement.nativeElement;
-    expect(compiled.querySelector('h1').textContent).toContain('Welcome to nortal-todo!');
+    expect(compiled.querySelector('h1').textContent).toContain('nortal-todo');
```

Comment:
nothing wrong with this approach in this context, but we could use data-attributes instead of looking for elements, id's or classes, as id's and classes are prone to change and then we'd need to update our tests, which can be cumbersome in the long run. With elements the issue is that if we now add another element, for example if we'd used h2 instead of h1 and now add another h2 level heading that we want to treat differently, then we'd need to rewrite our test selectors to distinguish between those two elements and we are back to the previous problem statement.

https://medium.com/@colecodes/test-your-dom-with-data-attributes-44fccc43ed4b